### PR TITLE
dashboard

### PR DESCRIPTION
**Pull request by Jaidenmagnan**

### CODE REVIEW

File: src/routes/(authed)/dashboard/+page.svelte
Comment (Nikhil-42) [inline src/routes/(authed)/dashboard/+page.svelte:1]: @Jaidenmagnan  I'm assuming you want to use the new UI

**Review by Nikhil-42 — CHANGES_REQUESTED** [review]

File: src/routes/(authed)/dashboard/+page.svelte
Comment (Jaidenmagnan) [reply src/routes/(authed)/dashboard/+page.svelte:1]: Yes I did not like the tailwind i setup in the last one so I am going to redo it on this commit.

**Review by Jaidenmagnan — COMMENTED** [review]
good changes

**Review by Nikhil-42 — APPROVED** [review]
Lgtm

**Pull request merged**